imagebuildah/build.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/containers/image/types"
 	"github.com/containers/storage"
 	"github.com/containers/storage/pkg/archive"
-	"github.com/cyphar/filepath-securejoin"
+	securejoin "github.com/cyphar/filepath-securejoin"
```

Comment:
just curious, did you add this or did VSCode do it for you?

Reply:
VSCode auto-formatted this for me. Happy to undo if you'd like :+1:

Reply:
No that's fine as is.  I just saw a similar change in another PR and was just curious.